source/dub/recipe/selection.d

```diff
@@ -16,7 +16,7 @@ public struct Selected
     public uint fileVersion;
 
     /// The selected package and their matching versions
-    public YAMLSelectedDependency[string] versions;
+    public YAMLSelectedDependency[PackageName] versions;
```

Comment:
@nordlow : Looks like this is where the error originate. Looks like a Configy bug, will look into it.

Reply:
Thank you!

Reply:
Have you made any progress on tracking down the Configy bug? If not what's the easiest way to circumvent this issue, @Geod24?

Reply:
You can use `string` for the moment yes.

Reply:
I've revert back to